resources/language/resource.language.nl_nl/strings.po

```diff
@@ -304,4 +307,4 @@ msgstr "Tijdelijke download folder"
 
 msgctxt "#30911"
 msgid "Update frequency in days"
-msgstr ""
+msgstr "Update frequentie in dagen"
```

Comment:
Should that not be one word? *Updatefrequentie*

Reply:
Yes.